python/nav/metrics/graphs.py

```diff
@@ -195,7 +201,7 @@ def get_simple_graph_url(metric_paths, time_frame="1day", title=None,
     :return: The URL that will generate the requested graph.
 
     """
-    if isinstance(metric_paths, basestring):
+    if isinstance(metric_paths, six.string_types):
```

Comment:
Here you import six instead of string_types directly. Any reason for it?

Reply:
I descided at some point that this was better, but it seems I did fix all cases